tests/assets/ansible/roles/rook/tasks/sles-15.yml

```diff
@@ -1,4 +1,18 @@
 ---
+- name: add repositories
+  vars:
+    repositories:
+      storage6: http://download.suse.de/ibs/SUSE/Products/Storage/6/x86_64/product/
```

Comment:
can we use SES7 here? I know we said we want to start with SES6, but things changed.

Reply:
Yes that is what I used when developing it.
I'll add a way to change version. Can we still merge this and I'll create a separate PR for this functionality ?